PasswordVerifier/src/com/company/PasswordVerifier.java

```diff
@@ -0,0 +1,81 @@
+public class PasswordVerifier {
+
+    public static void main(String[] args) {
+        String pass = "abdsdscdEF";
+        System.out.println("Password is : "+pass);
+        try {
+            System.out.println("Response is : "+verify(pass));
+        } catch (InvalidPasswordException e) {
+            System.out.println("Response is : "+e.getMessage());
+        }
+    }
+
+    public static String verify(String pass) throws InvalidPasswordException {
+        int count=0;
+        boolean isupper=false;
+        boolean isdigit = false;
+        if(pass==null || pass.trim().isEmpty() || pass.length()<8) {
+            throw new InvalidPasswordException("Password must contains atleast 8 characters");
+        }else {
```

Comment:
This is a minor thing but you have inconsistency in your spacing. There is a space before the `{` but no space after `}`. Some interviewers will feel that this shows a lack of attention to detail. Definitely don't want this to be the reason you lose out on a job opportunity 😀. 
You could also look at adding an auto code formatter to your code editor. These have become quite popular recently and make it so you never need to think about code formatting.